ladybug/futil.py

```diff
@@ -1,6 +1,18 @@
 """A collection of auxiliary funtions for working with files and directories."""
 import os
 import shutil
+import zipfile
+try:
+    import System.Net as sys
```

Comment:
This should move to ladybug[+] libraries. We don't want to include any .Net stuff in core libraries.

https://github.com/ladybug-tools/ladybug-grasshopper/tree/master/ladybug

Reply:
I am not sure if I understand.  Are you saying that we need to duplicate this exact same download function in the plus library of each and every CAD interface (so one copy in the plus library for Dyanmo and another one in the Grasshopper plus)?  I'm just thinking that this might lead to the exact thing that we are trying to avoid.  If the internet security protocol gets updated again, we will have to make this change in each and every plus library currently in existence (which might have Blender, Sketchup, etc. in the future). 
I'll do it if you are confident that this shouldn't be here but a copy in each plus library didn't seem right to me.

Reply:
Dynamo and Grasshopper (which are the only .NET plugins) can have shared libraries if needed. SketchUp and Blender are Ruby and Cpython. I agree that we don't want to duplicate code but that should be addressed in its own way and is another issue to be discussed. In any case we don't want to have .NET dependencies in ladybug core libraries which is a CPython library.

To address that we might need to create a new library which is built on top of ladybug and is used by all .NET libraries.

ladybug-grasshopper : includes files that will be used but Grasshopper plugin
ladybug-net: includes .NET specific scripts which are shared between .NET plugins
ladybug: the core ladybug library

Reply:
Ah, ok.  This is clearer to me now and I agree that it does not belong in the core.  I didn't realize that the use of the .NET libraries would not be possible in Sketchup or Blender so this function isn't nearly as reusable as I thought.
Based on your comment, I can see that we might eventually need a separate ladybug-net repo to house functions like this but this might be a bit overkill for the time being.  For now, what do you think about me pulling this out into it's own dotnet.py file that will just be copy/pasted in the ladybug-grasshopper and ladybug-dynamo repos?  We can house this function in there for now and, if we start realizing that there are a lot of different functions that need to be there, we can make it it's own repo.

Reply:
>  For now, what do you think about me pulling this out into it's own dotnet.py file that will just be copy/pasted in the ladybug-grasshopper and ladybug-dynamo repos? We can house this function in there for now and, if we start realizing that there are a lot of different functions that need to be there, we can make it it's own repo.

Sure. Sounds like a plan!